tests/testthat/test-stratify_data.R

```diff
@@ -4,18 +4,15 @@ test_that("stratify_data() produces consistent results", {
   library(readr)
 
   xs_data <-
-    read_rds("https://osf.io/download//n6cp3/")  %>%
-    as_pop_data() %>%
-    filter(Country == "Pakistan")
+    sees_pop_data_pk_100 %>%
+    filter(catchment == "aku")
```

Comment:
```suggestion
    sees_pop_data_pk_100
```

if there's only one catchment and we're stratifying by catchment below, it's not a very rigorous test.